Sources/Frontend/Commands/ScanCommand.swift

```diff
@@ -104,6 +107,7 @@ struct ScanCommand: FrontendCommand {
         configuration.apply(\.$workspace, workspace)
         configuration.apply(\.$project, project)
         configuration.apply(\.$fileTargetsPath, fileTargetsPath)
+        configuration.apply(\.$workingDirectory, workingDirectory)
```

Comment:
There are a few other pieces in `Configuration.swift` that need to be updated too. Just take a look at an existing configuration option as an example.